dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-3.0/com.ibm.websphere.appserver.jpaContainer-3.0.feature

```diff
@@ -0,0 +1,26 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jpaContainer-3.0
+visibility=public
+singleton=true
+kind=beta
+edition=core
+Subsystem-Name: Jakarta Persistence API Container 3.0
+IBM-ShortName: jpaContainer-3.0
+IBM-API-Package: jakarta.persistence; type="spec", \
+ jakarta.persistence.spi; type="spec", \
+ jakarta.persistence.criteria; type="spec", \
+ jakarta.persistence.metamodel; type="spec"
+IBM-App-ForceRestart: uninstall, \
+ install
+-features=com.ibm.websphere.appserver.jndi-1.0, \
+ com.ibm.websphere.appserver.classloading-1.0, \
+ com.ibm.websphere.appserver.optional.jaxb-2.2; ibm.tolerates:=2.3, \
+ com.ibm.websphere.appserver.jakarta.persistence-3.0, \
+ com.ibm.websphere.appserver.javax.annotation-1.3; apiJar=false, \
+ com.ibm.websphere.appserver.jdbc-4.2; ibm.tolerates:="4.3", \
+ com.ibm.websphere.appserver.transaction-1.2, \
+ com.ibm.websphere.appserver.javaeeCompatible-8.0
```

Comment:
should be javaeeCompatible-9.0

Reply:
Yeah I thought about that one, but I didn't spot a javaeeCompatible-9.0 or jakartaeeCompatible-9.0 feature.